graph-subscriptions-api/datasource/src/models.rs

```diff
@@ -142,20 +142,18 @@ impl RequestTicketOrderBy {
 }
 
 #[derive(Debug, Clone, PartialEq)]
-/// Stats pulled and aggergated/derived from queries made from users using The Graph Subscriptions; over all queried Subgraphs.
+/// Stats pulled and aggergated/derived from queries made from users using The Graph Subscriptions;
+/// over all queried Subgraphs and across all queried request tickets.
 /// The logs are pushed onto a kafka topic by The Graph Gateway.
-pub struct RequestTicketStat {
-    /// User-selected, friendly name of the request ticket. Part of the EIP-712 domain signed message.
-    /// Pulled directly from the kafka topic log from The Graph Gateway.
-    pub ticket_name: String,
-    /// Wallet address of the user who owns the request ticket.
+pub struct UserSubscriptionStat {
```

Comment:
The git diff view here is confusing and it may be easier to just view the file. But, I created a new type: `UserSubscriptionStat`, that has almost the same shape as the `RequestTicketStat`, but with the `ticket_name` and `queried_subgraphs_count` omitted.